docker/lite/build.sh

```diff
@@ -14,35 +14,41 @@ fi
 
 flavor=$1
 
+base_image=vitess/base
+lite_image=vitess/lite
+make_target=docker_base
 tag=latest
 if [[ -n "$flavor" ]]; then
   base_image=vitess/base:$flavor
+  lite_image=vitess/lite:$flavor
+  make_target=docker_base_$flavor
   tag=$flavor
 else
   echo "Flavor not specified as first argument. Building default image."
-  base_image=vitess/base
 fi
 
 # Abort if base image does not exist.
 if ! docker inspect $base_image &>/dev/null; then
-  echo "ERROR: Dependent image $base_image does not exist. Run 'make docker_base' to build it locally or 'docker pull $base_image' to fetch it from Docker Hub."
+  echo "ERROR: Dependent image $base_image does not exist. Run 'make $make_target' to build it locally or 'docker pull $base_image' to fetch it from Docker Hub (if we publish it)."
```

Comment:
Probably it's worth saying "if it's published" to not make people guess who is "we" here.